tests/testutils/ctx.go

```diff
@@ -64,7 +64,9 @@ func (d *dirDesc) cleanup() {
 		return
 	}
 	if err := os.RemoveAll(d.dir); err != nil && !os.IsNotExist(err) {
-		panic(fmt.Sprintf("Failed to remove temporary %s directory %q: %s", d.desc, d.dir, err))
+		//TODO: Fix race condition
```

Comment:
WIP: Sometimes ```Cleanup``` cannot remove workdir on kvm flavor. It appears when ```run-build``` is running.